24life/scripts/scripts.js

```diff
@@ -143,6 +155,15 @@ function decorateSpotifyLinks(main) {
  * @param {Element} main The container element
  */
 function buildAutoBlocks(main) {
+  const templateName = getMetadata('template');
+  if (templateName === 'newsletter') {
+    try {
+      buildHeroBlock(main);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('Auto Blocking failed', error);
+    }
+  }
```

Comment:
This is adding a second implementation of how templates are interpreted. 

If you look at your browser console, you will see that https://issue47-newsletter--24life--hlxsites.hlx.page/24life/templates/newsletter/newsletter.js is being requested and it cannot be found. 

Better implement newsletter.js if you really need it. Or there might be other ways to create a hero block from the document.

Reply:
I would instead use the `article-hero-video`. If it throws errors, make the block slightly more tolerant to missing metadata. Then you can remove the things from styles.css and scripts.js, making every page load a bit lighter.

Reply:
Working on the above.

Reply:
Fixed

Reply:
See comment in slack thread